package.json

```diff
@@ -13,10 +13,14 @@
   "devDependencies": {
     "@grafana/data": "7.0.6",
     "@grafana/runtime": "7.0.6",
-    "@grafana/toolkit": "7.0.6",
```

Comment:
Not sure we need this, I just gave it a go to see if that version of toolkit included a jest version > 24 (doesn't seem to)

Reply:
sounds like we should make sure it does in the future!